addons/website_forum/controllers/website_forum.py

```diff
@@ -171,6 +175,7 @@ def questions(self, forum=None, tag=None, page=1, filters='all', my=None, sortin
             'tag': tag,
             'filters': filters,
             'my': my,
+            'visible': visible,
```

Comment:
in `_prepare_open_forum_user` shouldn't we add "visible==True" in the domain for `user_question_ids`, `followed` and `user_answer_ids` ?
(otherwise they are still visible)